sfaira/train/summaries.py

```diff
@@ -125,6 +126,15 @@ def __init__(
         self.source_path = source_path
         self.summary_tab = None
 
+    @staticmethod
+    def create_topology(model_id):
+        model_class = model_id.split('_')[0]
+        organism = model_id.split('_')[1]
+        model_type = model_id.split('_')[3]
+        topo_id = model_id.split('_')[5]
+        topology = TOPOLOGIES[organism][model_class][model_type][topo_id]
+        return topology
```

Comment:
This fails with new model_id format. We have to go with the new format everywhere in this file I think; it's getting hairy trying to juggle both formats. I can go through the file and change the parsing everywhere and then we can merge this branch. What do you guys think? @le-ander @davidsebfischer

Reply:
that's a good idea, thanks for the offer! there is an updated version of this whole file in a jupyter notebook right now. we'll update the file in the next 1 or 2 days and then we can go through it.